tutorials/spacecraft_thrusters.md

```diff
@@ -0,0 +1,118 @@
+\page spacecraft_thrusters Spacecraft thrusters
+
+## Spacecraft Thrusters Model
+
+To enable a seamless transition of space robotics control and planning schemes from simulation to real, we introduce
+a spacecraft thrusters model in Gazebo. The model provides a very simple interface to a solenoid valve that controls
+the flow of gas to the thruster. The thruster model is as follows:
+
+- force output equals maxThrust when the command is 1
+- force output equals 0 when the command is 0
+- force output is modeled according to a duty cycle with a given frequency, and thrust output is maximum at the ON state of the duty cycle
+
+In short, if the duty cycle signal is high, the solenoid valve behaves as a fully-opened thruster, providing maximum thrust.
+If the duty cycle signal is low, the solenoid valve behaves as a fully-closed thruster, providing no thrust.
+
+## Setting up the SpacecraftThrusterModel plugin
+
+Here follows an example instance of the SpacecraftThrusterModel plugin in an SDF file:
+```xml
+    <plugin filename="gz-sim-spacecraft_thruster_model-system" name="gz::sim::systems::SpacecraftThrusterModel">
+      <linkName>thruster_0</linkName>
```

Comment:
We've been trying to improve consistency on the style we use for SDF parameters. In general,  we prefer `snake_case` unless there's a specific reason to use `camelCase`.

cc @bperseghetti

Reply:
Hi @bperseghetti , thanks for the PR ! I'll give it a look and do the required changes to the fuel model.

Regarding adding the 2D Model, I probably can also do that and add it to fuel as well, since I already have that model implemented anyway.

Regarding the test - I'll probably need to do those later since I really have no time for it now (other than maybe doing a very basic test). Let me fix the items above and then get to this.

Thanks for all the support!